src/steamship/cli/requirements_init_wizard.py

```diff
@@ -0,0 +1,20 @@
+import click
+
+import steamship
+
+
+def requirements_init_wizard():
+    click.secho(
+        "Steamship REQUIRES deployables to use requirements.txt to specify dependencies. You do not currently have a requirements.txt in this directory.",
```

Comment:
Unimportant / alternative phrasing idea:

"Steamship uses requirements.txt to specify dependencies. You..."

Reply:
I like this better